app/views/main.scala.html

```diff
@@ -30,6 +31,7 @@
         googleAnalyticsCalls: Seq[String] = Seq.empty,
         noGetHelp:Boolean = false,
         loggedIn:Boolean = false,
```

Comment:
`loggedIn` defaults to `false` here, and I think this will default to showing the logged out version of the timeout dialog.

Most of the pages/views in the service are likely to be shown only when logged in, so it's perhaps worth defaulting it to `true`. There's the odd few pages that may be shown when either logged in or logged out such as the first page of the journey.

Reply:
I found that `loggedIn` wasn't actually consumed anywhere in the `main` template - it's completely ignored. This explains why some templates had the "wrong" `loggedIn` value being passed in - it is of no consequence.

`req.RequestSupport.isLoggedIn` is the following that used to decide if the Sign out link should be displayed. I'm now also passing `req.RequestSupport.isLoggedIn` to `canSignOut` (which determines if to trigger timeout dialog)

This means the `loggedIn` parameter is both unnecessary and wrong for some templates. Is this something to remove as part of this ticket or is it better to make a note of it as tech debt for a later time?

Reply:
I'd fix it in this ticket if it's fairly straightforward. Tech debt has a habit of getting lost and unloved in the backlog.